.travis.yml

```diff
@@ -6,6 +6,10 @@ cache:
   directories:
     - $HOME/.composer/cache
 
+env:
+  - STABILITY=''
+  - STABILITY='--prefer-lowest'
```

Comment:
i tried to to optimize things to lower the build time and now is almost half time.
do you know some projects using the approach you suggested? to see exactly what do you mean